ducktape/command_line/defaults.py

```diff
@@ -15,7 +15,7 @@
 import os
 
 
-class ConsoleConfig(object):
+class ConsoleDefaults(object):
```

Comment:
Yeah good point - the fact that "we consider it to be internal" is not actually enforced